tests/test_search.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from legistar import base
+
+@pytest.fixture(scope="module")
```

Comment:
That seems like something we can do once we have more tests? Or is it a strong DataMade convention that fixtures should only go in conftest?

Reply:
I would argue that it's a convention. A centralized location for generic fixtures, like this one, lowers the barrier to writing additional test modules moving forward (because that fixture will be available for use in those modules). If there fixtures are truly specific to a test module, ok – but this one seems reusable beyond testing one method on the base scraper.

Reply:
I think it makes this thing a bit less readable, but conventions are important. I've made the change.

Is there anything else?

Reply:
This seems fine to me!